x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -47,34 +48,40 @@ import {
 import { DeleteSpaceAwareItemCheckModal } from '../../../../components/delete_space_aware_item_check_modal';
 
 interface Props {
-  details: any;
+  details: Record<string, any>;
   getNodeData: any;
   modelId?: string;
   updateElements: (nodeId: string, nodeLabel: string, destIndexNode?: string) => void;
   refreshJobsCallback: () => void;
 }
 
-function getListItems(details: object): EuiDescriptionListProps['listItems'] {
-  return Object.entries(details).map(([key, value]) => {
-    let description;
-    if (key === 'create_time') {
-      description = formatHumanReadableDateTimeSeconds(moment(value).unix() * 1000);
-    } else {
-      description =
-        typeof value === 'object' ? (
-          <EuiCodeBlock language="json" fontSize="s" paddingSize="s">
-            {JSON.stringify(value, null, 2)}
-          </EuiCodeBlock>
-        ) : (
-          value
-        );
+function getListItemsFactory(isServerless: boolean) {
+  return (details: Record<string, any>): EuiDescriptionListProps['listItems'] => {
+    if (isServerless) {
+      delete details.license_level;
     }
 
-    return {
-      title: key,
-      description,
-    };
-  });
+    return Object.entries(details).map(([key, value]) => {
+      let description;
+      if (key === 'create_time') {
+        description = formatHumanReadableDateTimeSeconds(moment(value).unix() * 1000);
```

Comment:
I'd prefer not to make this change in this PR as it is out of scope.
I agree that `useFieldFormatter` would be better to use. `formatHumanReadableDateTimeSeconds` is used 22 times in the ML code base, so I think it would be good to change them all in one PR.